### ISSUE TITLE
howto

### ISSUE THREAD

**Issue by sbakkis** [trapexit/mergerfs#430]
Gents,

I am probably a noob in most respects when it comes to linux. I love it but have no time getting to expert level I'm afraid.

Any chance a small getting a howto for mergerfs-tools? I am running mergerfs + SnapRAID in OMV, and I think it is fantastic. Much better than the Raid/ZFS solutions I've tried earlier, for my use case (media storage for plex/kodi).

The reason I want a howto is so that I can run balance as content is very unevenly distributed on my 5x8TB IronWolf drives.

All the work you put in is much appreciated.

EDIT : I realize this should be in mergerfs-tools, and not mergerfs. Sorry. Let me know if you want me to delete and create a new in the correct place.

Best,
Stian

**Comment by bauruine** [comment]
There is a "howto" https://github.com/trapexit/mergerfs-tools#mergerfsbalance

tl;dr `mergerfs.balance /path/to/mergerfs/mount`

**Comment by sbakkis** [comment]
Thank you for your response.

Yes, I have seen it. Is it just downloading the zip and put it on the server in some folder, then run commands? I don't want to install full git tools on the server, if possible.

**Comment by bauruine** [comment]
Just unzip, cd to the src folder and then `./mergerfs.dedup /path/`

**Comment by trapexit** [comment]
There is no proper install for the tools at the moment but they are all self contained. Just download the file from git and you can use it (so long as python is installed).
